.github/pull_request_template.md

```diff
@@ -1,29 +1,28 @@
-<!-- Make sure that your title neatly summarizes the proposed changes -->
+<!-- 
+Thanks for opening a pull request to Prefect! We've got a few requests to help us review contributions:
 
-### Summary
-<!-- Provide a short overview of the change and the value it adds -->
-
-### Steps Taken to QA Changes
```

Comment:
Speaking from the POV of a managing a release this section has been very helpful. Maybe we could at least add a checkbox saying that they did more than just write code? Like "I tested this manually"?

Reply:
Well, right now we're seeing that section often contain "Wrote unit tests". I'm hoping that by forcing an example we'll get a real blurb instead of that. I guess there's no proof that they ran the example though. We can add a checkbox:

- [ ] The change in behavior introduced by this pull request was verified outside of automated tests

but I wonder if it's premature.

Reply:
That's fair, I'm just wanting to make sure that running this example doesn't fall to whoever is doing the release as that can already take a fair amount of time to just get the proverbial cats herded.